plugin/src/test/testProject/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+import com.yelp.codegen.plugin.GenerateTaskConfiguration
+
+buildscript {
+  repositories {
+    maven {
+      url = uri("../../../build/localMaven")
+    }
+    jcenter()
```

Comment:
If you change this into:

```kotlin
    jcenter {
      content {
        excludeGroup("com.yelp.codegen")
      }
    }
```

we will be sure we're actually getting the plugin only from MavenLocal

Reply:
Nice ! Will do :+1: